samples/NativeIISSample/Startup.cs

```diff
@@ -14,8 +14,15 @@ namespace NativeIISSample
 {
     public class Startup
     {
+        private readonly IAuthenticationSchemeProvider _authSchemeProvider;
+
+        public Startup(IAuthenticationSchemeProvider authSchemeProvider = null)
+        {
+            _authSchemeProvider = authSchemeProvider;
```

Comment:
Nit: why move the AuthSchemeProvider out of Configure?

Reply:
It's not available in outofproc so I've made it optional